SynCrtSock.pas

```diff
@@ -1843,6 +1845,11 @@   THttpServer = class(THttpServerGeneric)
     // a THttpServerResp thread is created for handling this THttpServerSocket
     property Sock: TCrtSocket read fSock;
   published
+    /// flag to disable HTTP/1.1 keep alive features and fall back to
+    // HTTP/1.0 compatibility
+    // Only threads from thread pool will be used to process requests

```

Comment:
missing traiiling "-" character in comment

Reply:
Change committed. Did I understand you correctly?